drned/drned/__init__.py

```diff
@@ -1,10 +1 @@
-"""@package drned
-Documentation for this module.
-
-More details.
-"""
 from .device import Device
-from .node import Node
```

Comment:
this leads to implicit reexports that lead to problems with type annotations further down the line...
e.g. schema is being still imported from previous file despite explicit export being removed here...

Reply:
No, `schema` is not imported in `device`; and I'm failing to see any (potential or actual) problems with type annotations, the module or its types are not used except in `test_coverage`.